### ISSUE TITLE
Provide more context in the documentation

### ISSUE THREAD

**Issue by sdeleon28** [oblitum/swift-ycm-core#2]
I don't know Swift and want to install this plugin precisely to learn it. The build section looks a little obscure to me. Can you provide a little more context in the README file? Perhaps include snippets with the bash commands you would run if you were to install this plugin.

**Comment by lui213141** [comment]
I would have to agree with sdeleon28.  I am extremely interested in this plugin because I cannot seem to get any autocompletion plugin working similarly to Xcode's autocompletion for swift, yet the documentation for installation is extremely obscure.  If you could simplify the installation and provide specific commands this would be a much better plugin.

**Comment by Wes974** [comment]
I agree with them. I'm also very interested in this plugin but can't figure out what to do to install it. Please, can you provide more context ? Thank you

**Comment by oblitum** [comment]
This code will stay stale sorry, I don't have interest in Swift at the moment and foreseeable future and can't say whether the build instructions are still valid.
